X00TD.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Ken_kaneki_69 (Paras Dhiman)",
+        "oem": "Asus",
+        "device": "Asus Zenfone Max Pro M1",
+        "filename": "crDroidAndroid-12.1-20220416-X00TD-v8.4.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/X00TD/8.x/crDroidAndroid-12.1-20220416-X00TD-v8.4.zip/download",
+        "timestamp": 1650131889,
+        "md5": "7aeb8195a78e3266212e1f556844d2a5",
+        "sha256": "240b55e7f86ce714259b04f1235cb5e7b45728519c4c727cbbd1a0618d365f53",
+        "size": 1099808016,
+        "version": "8.4",
+        "buildtype": "Monthly",
+        "forum": "https://t.me/mst_roms_X00T",
```

Comment:
this needs to be forum url (like XDA or similar) with our forum template
https://raw.githubusercontent.com/crdroidandroid/crdroid_features/12.1/ForumPostTemplate